src/fu-tool.c

```diff
@@ -2422,6 +2424,12 @@ main (int argc, char *argv[])
 		{ "force", '\0', 0, G_OPTION_ARG_NONE, &force,
 			/* TRANSLATORS: command line option */
 			_("Override plugin warning"), NULL },
+		{ "ignore-checksum", '\0', 0, G_OPTION_ARG_NONE, &ignore_checksum,
+			/* TRANSLATORS: command line option */
+			_("Ignore firmware checksum failures"), NULL },
+		{ "ignore-vid-pid", '\0', 0, G_OPTION_ARG_NONE, &ignore_vid_pid,
```

Comment:
these should be added to bash completion

Reply:
I think there should be some kind of swear-jar for every time I forget to add these. My apologies.